sasdata/dataloader/loader.py

```diff
@@ -363,14 +364,26 @@ def associate_file_reader(self, ext: str, loader) -> bool:
         """
         return self.__registry.associate_file_reader(ext, loader)
 
-    def load(self, file: str, format: Optional[str] = None) -> Union[list, Exception]:
+    def load(self, file_path_list: Union[List[str], str],
+             format: Optional[Union[List[str], str]] = None
+             ) -> Union[List[Union[Data1D, Data2D]], Exception]:
         """
-        Load a file
-        :param file: file name (path)
+        Load a file or series of files
+        :param file_path_list: String representations of any number of file paths. This can either be a list or a string
         :param format: specified format to use (optional)
         :return: DataInfo object
         """
-        return self.__registry.load(file, format)
+        if isinstance(file_path_list, str):
```

Comment:
What if you are called with a `pathlib.Path` object? Though that might have other issues.

I've been moving away from interfaces that accept either a singleton or a list toward separate functions for singletons and lists. However, in the list case you may want to accept a single format or a sequence of formations of the same length.

You can loop over `zip(file_path_list, format_list)` rather than using an index into `format_list`.